docs/live-streaming.md

```diff
@@ -0,0 +1,157 @@
+
+Live streaming with SRT - guidelines
+====================================
+
+SRT is primarily created for live streaming. Before you use it, you must keep
+in mind that Live Streaming is a process with its own rules, of which SRT
+fulfills only and exclusively the transmission part. Live Streaming process
+consists of more parts, though.
+
+
+Transmitting MPEG TS binary protocol over SRT
+=============================================
+
+MPEG-TS is the most important protocol predicted to be sent over internet using
+SRT and the main reason of initiating this project. SRT obviously isn't limited
+to MPEG-TS, howver it still is limited in use to any data transmission that can
+be called Live Streaming - at least if you use the Live mode (which is SRT
+default).
+
+MPEG-TS consists of single units of 188 bytes. By having `188*7` we get the
+size of 1316, which is the maximum multiplicity of 188 that results still less
+than 1500, the standard MTU size in UDP transport, which is the underlying
+transport layer for SRT. The headers for Ethernet, IP and UDP protocol occupy
```

Comment:
28 is only IP header (20) + UDP (8), ethernet headers are not part of IP (not UDP) MTU

Reply:
"IP (not UDP) MTU"... actually ethernet MTU like Adam said.

Reply:
Ok, Ethernet removed from this "list".